assets/css/layout/_footer.scss

```diff
@@ -0,0 +1,158 @@
+// Footer styles.
+
+footer {
+  background: $navy-light;
+  color: $white;
+  margin-top: rem(30);
+  padding: rem(16) rem(32);
+
+  .footer-content {
+    align-items: baseline;
+    display: flex;
+    flex-direction: column;
+    font-size: rem(16);
+    line-height: rem(29);
+    max-width: $max-width-margins;
+
+    h2 {
+      font-size: rem(16);
+      font-weight: bold;
+    }
+
+    .contact-info {
+      width: 100%;
+
+      a {
```

Comment:
Insufficient contrast for these links:

<img width="720" alt="Screen Shot 2020-03-30 at 10 12 59 AM" src="https://user-images.githubusercontent.com/605361/77935734-15bf0d00-726f-11ea-8287-d5935e3d758a.png">